.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "Programmer-Art-Plus"]
 	path = Programmer-Art-Plus
 	url = https://github.com/Crackers0106/Programmer-Art-Plus
+[submodule "golden-days-base"]
+	path = golden-days-base
+	url = https://github.com/AlexTMjugador/golden-days-base.git
```

Comment:
Can't you configure the workflow to use the official repository?

Reply:
I used my cloned repo to avoid setting different subfolder paths in the matrix and make things more consistent, but given that we cannot use matrices here, we might as well use that to our advantage and slightly tweak the paths in this case anyway.